metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py

```diff
@@ -150,9 +150,12 @@ def extract_lineage(
     ) -> List[MetadataChangeProposalWrapper]:
         mcps: List[MetadataChangeProposalWrapper] = []
 
+        assert table.dataset
```

Comment:
Is this assert safe? Please confirm there's a catch-all except that won't fail the entire connector above this method

Reply:
it should be safe, but i've tweaked it just in case